### ISSUE TITLE
Add information on how to check current cluster size under Linux

### ISSUE THREAD

**Issue by MateusRodCosta** [hacks-guide/Guide_3DS#2391]
**Pages with issue(s)**

https://3ds.hacks.guide/formatting-sd-(linux)

**Description of the issue(s)**

It is actually possible to determine what is the current cluster size of a FAT32 partition under Linux, it might be useful to add that information to guide.

The command is:

`$ fsck.fat -n -v /dev/<SD card device>`

`-n` here means do a check without changing anything and `-v` for verbose.

The important part is this section:

```
Media byte 0xf8 (hard disk)
       512 bytes per logical sector
     32768 bytes per cluster
        64 reserved sectors
```

So it's possible to see the SD card is formatted with 32 kb cluster size.

**Comment by lifehackerhansol** [comment]
Do you think it's worth the double check? i.e. Our given mkdosfs command should have it formatted correctly.

We don't double-check for Windows either, so if this were to be added, it should be applied to the other operating systems as well.

**Comment by MateusRodCosta** [comment]
Hi, sorry I forgot about this.

> Do you think it's worth the double check?

It might not be useful to immediately check the cluster size after formatting.

However it might be useful to check it if you formatted with some other tool or have an old install and you are considering whether you should re-format the SD card or not (moving all files to a PC and back might be very slow, depending on size of card and speed).
In that case you would check.

> We don't double-check for Windows either, so if this were to be added, it should be applied to the other operating systems as well.

I'm not very familiar with how to do this on Windows, but I would guess either the properties window or the Disk management tool could easily show this info.

It's a bit unfortunate that GNOME Disks on Linux kinda hides it  (although it might be exposed via udisks)

**Comment by lifehackerhansol** [comment]
Coming back to this now, I feel as though it doesn't really need to be added prior to the format.

Cluster size isn't the only reason users format their SD cards, there's a plethora of other reasons one should to do it, an example being a corrupted file system. And in general it might fix unforeseen issues regardless. We shouldn't add a "if you're just here for this, then you may as well check before starting" because this is going to add confusion for any non-technical folk out there (though if a user is on Linux, then maybe they're already technically capable...)

TLDR: if someone is considering to format their SD card, just do it anyway regardless of whether the condition of the SD is already good or not. As you say it may be slow, but I don't really see that as a huge issue.
